[2025-03-19T22:24:34.864+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-19T22:24:35.718+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_taskflow_Api_v2.greet scheduled__2025-02-08T00:00:00+00:00 [queued]>
[2025-03-19T22:24:36.056+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_taskflow_Api_v2.greet scheduled__2025-02-08T00:00:00+00:00 [queued]>
[2025-03-19T22:24:36.062+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 6
[2025-03-19T22:24:36.881+0000] {taskinstance.py:2890} INFO - Executing <Task(_PythonDecoratedOperator): greet> on 2025-02-08 00:00:00+00:00
[2025-03-19T22:24:37.008+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_with_taskflow_Api_v2', 'greet', 'scheduled__2025-02-08T00:00:00+00:00', '--job-id', '3252', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_taskflow_api.py', '--cfg-path', '/tmp/tmp3plt0qjm']
[2025-03-19T22:24:37.047+0000] {standard_task_runner.py:105} INFO - Job 3252: Subtask greet
[2025-03-19T22:24:37.012+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=912) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-19T22:24:37.072+0000] {standard_task_runner.py:72} INFO - Started process 934 to run task
[2025-03-19T22:24:38.165+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_with_taskflow_Api_v2.greet scheduled__2025-02-08T00:00:00+00:00 [running]> on host 63fd4b143e0a
[2025-03-19T22:24:40.817+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_with_taskflow_Api_v2' AIRFLOW_CTX_TASK_ID='greet' AIRFLOW_CTX_EXECUTION_DATE='2025-02-08T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-08T00:00:00+00:00'
[2025-03-19T22:24:40.890+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-19T22:24:41.080+0000] {logging_mixin.py:190} INFO - Hello JohnDoe, you are 19 years old.
[2025-03-19T22:24:41.083+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-19T22:24:41.869+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-19T22:24:41.889+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=dag_with_taskflow_Api_v2, task_id=greet, run_id=scheduled__2025-02-08T00:00:00+00:00, execution_date=20250208T000000, start_date=20250319T222435, end_date=20250319T222441
[2025-03-19T22:24:46.742+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-19T22:24:47.462+0000] {standard_task_runner.py:217} INFO - Process not found (most likely exited), stop collecting metrics
[2025-03-19T22:24:48.650+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
